code/modules/network/commands/directory_commands.dm

```diff
@@ -0,0 +1,258 @@
+
+//////////////////////////////
+///// Directory Commands /////
+//////////////////////////////
+
+// An obvious starting point for clueless players. Hints to the next useful command."
+
+/datum/network_command/info
+	trigger = "help"
+	info = "Usage: \"help -{query}\". Gives you some information about the queried command. You can use \"probe -c\" for a list of available commands."
+
+/datum/network_command/info/execute(datum/network/N, list/A, obj/item/device/hacktool/H)
+	..()
+	if(!A)
+		feedback = "[info]"
+		feed(H)
+		security(N, "noob", H)
+		return
+	for(var/datum/network_command/C in N.commands)
+		if(C.trigger == A[1])
+			feedback = "[C.info]"
+			security(N, "noob", H)
+			feed(H)
+			return
+	feedback = "Invalid Argument: \"[A[1]]\""
+	feed(H)
+	return
```

Comment:
trailing return, remove it please.
there's quite a few of these, I won't comment them all (unless requested)